src/main/scala/org/globalforestwatch/util/IntersectGeometry.scala

```diff
@@ -0,0 +1,69 @@
+package org.globalforestwatch.util
+
+import geotrellis.vector.{Geometry, Polygon, Point}
+import org.locationtech.jts.geom.Coordinate
+
+object IntersectGeometry {
+
+  def getIntersecting1x1Grid(geometry: Geometry): IndexedSeq[Polygon] = {
```

Comment:
If it's always going to be the same grid, should we just save it to a file and load it instead of generating it across each node every time? In fact there's already file that's used for geostores.